### PR TITLE
Edits/comments to therapeutics manuscript

### PR DESCRIPTION
**Pull request by SiminaB**
### Description of the proposed additions or changes

Look over the entire manuscript. I suggested changing the title to take out "prophylactics," made some changes throughout the manuscript related to this, and added more effect sizes/p-values/confidence intervals from the cited papers to make it a bit quantitative.

### Suggested reviewers (optional)

@rando2 @agitter @cgreene 

### Checklist

- [ X] Text is formatted so that each sentence is on its own line.

- [ X] Pre-prints cited in this pull request have a GitHub issue opened so that they can be reviewed.

### CODE REVIEW

File: content/20.pharmaceuticals.md [hunk 1]
```diff
@@ -557,12 +559,12 @@ Additionally, dexamethasone is a widely available and affordable medication, rai
 
 The results of the RECOVERY trial's analysis of dexamethasone suggest that this therapeutic is effective primarily in patients who had been experiencing symptoms for at least seven days and patients who were not breathing independently [@doi:10.1056/NEJMoa2021436].
 A meta-analysis that evaluated the results of the RECOVERY trial alongside trials of other corticosteroids, such as hydrocortisone, similarly concluded that corticosteroids may be beneficial to patients with severe COVID-19 who are receiving oxygen supplementation [@doi:10.1053/j.jvca.2020.11.057].
-Thus, it seems likely that dexamethasone is useful for treating inflammation associated with immunopathy or cytokine release syndrome (CRS).
+Thus, it seems likely that dexamethasone is useful for treating inflammation associated with immunopathy or cytokine release syndrome (CRS). <!--Should have a brief description of CRS here and in the future, a reference to the pathogenesis paper-->
```
Comment (agitter) [inline content/20.pharmaceuticals.md:562]: We can add the pathogenesis cross-reference now: `[@individual-pathogenesis]`

File: content/20.pharmaceuticals.md [hunk 2]
```diff
@@ -606,7 +608,7 @@ Both classical and trans-signaling can occur through three independent pathways:
 These signaling pathways are involved in a variety of different functions, including cell type differentiation, immunoglobulin synthesis, and cellular survival signaling pathways, respectively [@doi:10.3389/fimmu.2016.00604].
 The ultimate result of the IL-6 cascade is to direct transcriptional activity of various promoters of pro-inflammatory cytokines, such as IL-1, TFN, and even IL-6 itself, through the activity of NF-κB [@doi:10.3389/fimmu.2016.00604].
 IL-6 synthesis is tightly regulated both transcriptionally and post-transcriptionally, and it has been shown that viral proteins can enhance transcription of the IL-6 gene by strengthening the DNA-binding activity between several transcription factors and IL-6 gene-cis-regulatory elements [@doi:10.2492/inflammregen.33.054].
-Therefore, drugs inhibiting the binding of IL-6 to IL-6R&alpha; or sIL-6R&alpha; are of interest for combating the hyperactive inflammatory response characteristic of CRS and CSS.
+Therefore, drugs inhibiting the binding of IL-6 to IL-6R&alpha; or sIL-6R&alpha; are of interest for combating the hyperactive inflammatory response characteristic of CRS and CSS. <!--Aren't these the same thing?-->
```
Comment (rando2) [inline content/20.pharmaceuticals.md:611]:
```suggestion
Therefore, drugs inhibiting the binding of IL-6 to IL-6R&alpha; or sIL-6R&alpha; are of interest for combating the hyperactive inflammatory response characteristic of CRS/CSS. <!--Aren't these the same thing?-->
```
It's not entirely clear to me whether CSS is a coherent concept, but it seems like the papers that refer to it don't use CRS so I would guess they are the same thing?

File: content/20.pharmaceuticals.md [hunk 3]
```diff
@@ -633,43 +635,44 @@ Therefore, patients who would have been eligible for TCZ prior to this arbitrary
...
+The study reported improvement of survival outcomes after the addition of TCZ to the SOC regime, with 11 of 23 patients (47.8%) admitted prior to March 13th dying compared to 2 of 62 (3.2%) admitted afterwards (HR=0.035; 95% CI, 0.004 to 0.347; _p_ = 0.004).
 They also reported a reduced progression to mechanical ventilation in the TCZ group.
 However, this study also holds a significant limitation: the time delay between the two groups means that knowledge about how to treat the disease likely improved over this timeframe as well.
 All the same, the results of these observational retrospective studies provide support for TCZ as a pharmaceutical of interest for follow-up in clinical trials.
 
-In addition to the retrospective observational studies, other analysis have utilized a retrospective case-control design to match pairs of patients with similar baseline characteristics, only one of whom received TCZ for COVID-19.
+Other retrospective analyses have utilized a case-control design to match pairs of patients with similar baseline characteristics, only one of whom received TCZ for COVID-19. <!--Edited because case-control studies are a type of retrospective design.-->
```
Comment (rando2) [inline content/20.pharmaceuticals.md:643]:
```suggestion
Other retrospective analyses have utilized a case-control design to match pairs of patients with similar baseline characteristics, only one of whom received TCZ for COVID-19.
```

File: content/20.pharmaceuticals.md [hunk 4]
```diff
@@ -633,43 +635,44 @@ Therefore, patients who would have been eligible for TCZ prior to this arbitrary
...
 Taken together, these findings suggest that future clinical trials of TCZ may want to include intubation as an endpoint.
 However, these studies should be approached with caution, not only because of the small number of patients enrolled and the retrospective design, but also because they performed a large number of statistical tests and did not account for multiple hypothesis testing.
-These last findings highlight the need to search for a balance between impairing a harmful immune response, such as the one generated during CRS and CSS, and preventing the worsening of the clinical picture of the patients by potential new viral infections.
+In general, caution must be exercised when interpreting subgroup analyses after a primary combined endpoint analysis. <!--Need to be careful about overinterpreting subgroup analyses. Combined endpoints (of 2 events, like death OR ICU admission) are also problematic, though they are often used to increase power.-->
```
Comment (rando2) [inline content/20.pharmaceuticals.md:651]:
```suggestion
In general, caution must be exercised when interpreting subgroup analyses after a primary combined endpoint analysis.<!--Need to be careful about overinterpreting subgroup analyses. Combined endpoints (of 2 events, like death OR ICU admission) are also problematic, though they are often used to increase power.-->
```

File: content/20.pharmaceuticals.md [hunk 5]
```diff
@@ -633,43 +635,44 @@ Therefore, patients who would have been eligible for TCZ prior to this arbitrary
...
+These last findings highlight the need to search for a balance between impairing a harmful immune response, such as the one generated during CRS and CSS, and preventing the worsening of the clinical picture of the patients by potential new viral infections. <!--Aren't CRS and CSS the same thing?-->
```
Comment (rando2) [inline content/20.pharmaceuticals.md:652]:
```suggestion
These last findings highlight the need to search for a balance between impairing a harmful immune response, such as the one generated during CRS/CSS, and preventing the worsening of the clinical picture of the patients by potential new viral infections.
```

File: content/20.pharmaceuticals.md [hunk 6]
```diff
@@ -633,43 +635,44 @@ Therefore, patients who would have been eligible for TCZ prior to this arbitrary
...
 An increased risk of bacterial infection was also identified in a systematic review of the literature, based on the unadjusted estimates reported [@doi:10.2139/ssrn.3642653].
 TCZ administration to COVID-19 patients is not without risks, may introduce additional risk of developing secondary infections, and should be approached especially cautiously for patients who have latent viral infections.
 
-In summary, approximately 25% of coronavirus patients develop ARDS, which is caused by an excessive early response of the immune system which can be a component of CRS [@doi:10.1016/j.chest.2020.06.006] and CSS [@doi:10.1186/s12967-020-02339-3].
+In summary, approximately 25% of hospitalized coronavirus patients develop ARDS, which is caused by an excessive early response of the immune system which can be a component of CRS [@doi:10.1016/j.chest.2020.06.006] and CSS [@doi:10.1186/s12967-020-02339-3]. <!--Are CRS and CSS different from each other?--> <!--ARDS is only developed by 25% of *hospitalized* patients, not all patients. Should include a reference for that - I found this https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7441837/, which estimates this at 33%.-->
```
Comment (rando2) [inline content/20.pharmaceuticals.md:675]:
```suggestion
In summary, approximately 33% of hospitalized COVID-19 patients develop ARDS [@doi:10.1186/s13054-020-03240-7], which is caused by an excessive early response of the immune system which can be a component of CRS/CSS [@doi:10.1016/j.chest.2020.06.006; @doi:10.1186/s12967-020-02339-3].
```

File: content/20.pharmaceuticals.md [hunk 7]
```diff
@@ -685,7 +688,7 @@ Clinical trials that are in progress are likely to provide additional insight in
 Monoclonal antibodies have revolutionized the way we treat human diseases.
 They have become some of the best-selling drugs in the pharmaceutical market in recent years [@doi:10.1186/s12929-019-0592-z].
 There are currently 79 FDA approved mAbs on the market, including antibodies for viral infections (e.g. Ibalizumab for HIV and Palivizumab for RSV) [@doi:10.1186/s12929-019-0592-z; @doi:10.1146/annurev-immunol-032712-095916].
-Virus-specific nAbs commonly target viral surface glycoproteins or host structures, thereby inhibiting viral entry through receptor binding interference [@doi:10.3389/fmicb.2017.02323; @doi:10.1080/21645515.2017.1337614].
+Virus-specific nAbs commonly target viral surface glycoproteins or host structures, thereby inhibiting viral entry through receptor binding interference [@doi:10.3389/fmicb.2017.02323; @doi:10.1080/21645515.2017.1337614]. <!--Are these all monoclonal? If not, the flow is a bit off-->
```
Comment (rando2) [inline content/20.pharmaceuticals.md:691]: @nilswellhausen do you know the answer to this?

File: content/20.pharmaceuticals.md [hunk 8]
```diff
@@ -769,7 +772,7 @@ As more evidence becomes available, the potential for existing and novel therapi
 **Mechanism of Action for Potential Therapeutics**
 Potential therapeutics currently being studied can target the SARS-CoV-2 virus or modify the host environment through many different mechanisms.
 Here, the relationship between the virus and several therapeutics described above are visualized.
-](images/N001-LifeCyclePlusDrugs.png){#fig:therapeutics secno=1}
+](images/N001-LifeCyclePlusDrugs.png){#fig:therapeutics secno=1} <!--Step #5 is not labeled-->
```
Comment (rando2) [inline content/20.pharmaceuticals.md:775]: @nilswellhausen would you be able to add a 5 into this? Sorry I missed it at review!

File: content/20.pharmaceuticals.md [hunk 9]
```diff
@@ -782,7 +785,7 @@ The potential for other drugs, such as tocilizumab, to reduce recovery time rema
 
 One additional concern is that the presentation of COVID-19 appears to be heterogeneous across the lifespan.
 Many adult cases, especially in younger adults, present with mild symptoms or even asymptomatically, while others, especially in older adults, can be severe or fatal.
-In children, the SARS-CoV-2 virus can present as two distinct diseases, COVID-19 or MIS-C.
+In children, the SARS-CoV-2 virus can present as two distinct diseases, COVID-19 or MIS-C. <!--Should give brief explanation and reference for MIS-C.-->
```
Comment (rando2) [inline content/20.pharmaceuticals.md:788]:
```suggestion
In children, the SARS-CoV-2 viral infection can present either as a respiratory illness comparable to COVID-19 or as an inflammatory condition, known as multisystem inflammatory syndrome in children, for which presentation is similar to Kawasaki Disease [@doi:10.1038/s41390-020-1053-9].
```

**Review by rando2 — APPROVED** [review]
@SiminaB there's a lot here -- would you be able to look through these responses? I need to reach out to some of the original authors to get their input so there are several To Dos for me  (and @nilswellhausen) in here as well.

File: content/20.pharmaceuticals.md [hunk 10]
```diff
@@ -48,47 +50,48 @@ Similarly, the first case of SARS was reported in November 2002 in the Guangdong
...
 IFNs and Ribavirin have shown promise in _in vitro_ analyses of MERS, but their clinical effectiveness remains unknown [@doi:10.1038/nrmicro.2016.81].
 Therefore, only limited pharmaceutical advances from prior HCoV outbreaks can be adopted to COVID-19.
-Importantly, though, prior analyses of the virological and pathogenic properties of SARS-CoV-1 and MERS-CoV provide a strong foundation for the development of hypotheses about SARS-CoV-2 that have served to accelerated the development and identification of potential prophylactic and therapeutic approaches.
+Importantly, though, prior analyses of the virological and pathogenic properties of SARS-CoV-1 and MERS-CoV have provided a strong foundation for the development of hypotheses about SARS-CoV-2 that have served to accelerated the development and identification of potential therapeutic approaches.
```
Comment (rando2) [inline content/20.pharmaceuticals.md:75]:
```suggestion
Importantly, though, prior analyses of the virological and pathogenic properties of SARS-CoV-1 and MERS-CoV have provided a strong foundation for the development of hypotheses about SARS-CoV-2 that have served to accelerated the development and identification of potential therapeutic approaches.
Even during the early months of the pandemic, a number of studies have emerged investigating the potential effects of drugs with mechanisms suggested based on prior understanding of coronavirus-host interactions.
Initially, these were primarily observational studies, which simply compare groups of patients who did and did not receive a treatment to determine whether it may have an effect.
However, these studies are subject to confounding, and randomized controlled trials are the standard means of assessing the effects of medications.
In randomized controlled trials, patients are prospectively and randomly assigned to treatment conditions, allowing for much stronger interpretations to be drawn; however, data from these trials takes much longer to collect.
Both have proven to be important sources of information during the COVID-19 crisis, but as more results are released from randomized controlled trials, more definitive answers are becoming available about various proposed therapeutics.
```

**Review by rando2 — COMMENTED** [review]
Address obs vs RCT comments that were in ARBs section

File: content/20.pharmaceuticals.md [hunk 11]
```diff
@@ -835,7 +838,7 @@ While still nascent, work in this area is promising.
 Over the longer term, this approach and others may lead to the development of novel therapeutics specifically for COVID-19 and SARS-CoV-2.
 <!-- This section could be updated with most recent information on drug discovery efforts -->
 
-#### Conclusions
+#### Conclusions <!--I’m not sure we need this section. We’ve stated these things before in this manuscript and the section reads as how useful it’s been to learn from SARS-CoV-2 and MERS-CoV – it has been useful, but I don’t think we need that as the overall conclusion…-->
```
Comment (rando2) [inline content/20.pharmaceuticals.md:841]: I feel a little weird ending on the potential avenues section, if anyone has any ideas for how to flesh out conclusions, that would be awesome!

Comment (RLordan) [reply content/20.pharmaceuticals.md:841]: I think the mini conclusion that is there is sufficient. As Simina says we conclude a lot within the manuscript.

**Review by RLordan — APPROVED** [review]
Minor edits and a comment or two

File: content/20.pharmaceuticals.md [hunk 12]
```diff
@@ -215,22 +216,22 @@ Patients were randomized in equal proportions into four experimental conditions
...
+Therefore, the arguments put forward in defense of remdesivir do not necessarily seem robust in light of the large sample size used in the Solidarity trial, especially since the broad international nature of the Solidarity clinical trial, which included countries with a wide range of economic profiles and a variety of healthcare systems, provides a much-needed global perspective in a pandemic [@doi:10.1056/NEJMe2034294]. <!--This can be good and bad: For instance, the effect of remdesivir can be different in different countries, based on the way the SOC is applied (including general care conditions). Not sure if this is discussed in any of the papers.-->
+On the other hand, only 62% of patients in the Solidarity trial were randomized on the day of admission or one day afterwards [@doi:10.1056/NEJMoa2023184], and concerns have been raised that differences in disease progression could influence the effectiveness of remdesivir [@doi:10.1056/NEJMe2034294]. 
+Furthermore
```
[the remark this answers is not in the document]
Comment (SiminaB) [reply content/20.pharmaceuticals.md:230]: Not sure why it shows up as an edited sentence...

File: content/20.pharmaceuticals.md [hunk 13]
```diff
@@ -414,34 +416,34 @@ A patient with systemic lupus erythematosus developed a prolonged QT interval th
...
 Another limitation of the study that the authors acknowledge was that the HCQ treatment began, on average, at a 16-day delay from the symptom onset.
 The fact that this study was open-label and lacked a placebo limits interpretation, and additional analysis is required to determine whether HCQ reduces inflammatory response.
-Therefore, despite some potential areas of investigation identified in _post hoc_ analysis, this study cannot be interpreted as providing support for HCQ as a therapeutic against COVID-19.
+Therefore, despite some potential areas of investigation identified in _post hoc_ analysis, this study cannot be interpreted as providing support for HCQ as a therapeutic against COVID-19. <!--I would think the main reason for this is that the negative seroconversion and the median time to symptom alleviation are very close between groups, so that alone - without the other issue - would be enough to lead to this interpretation.-->
```
[the remark this answers is not in the document]
Comment (SiminaB) [reply content/20.pharmaceuticals.md:436]: Sorry for being unclear! I meant that this study was a null study ("The primary endpoint of the study was the negative conversion rate of SARS-CoV-2 in 28 days, and the investigators found no difference in this parameter between the groups... The median time to symptom alleviation was similar across different conditions (19 days in HCQ+SOC vs. 21 days in SOC "), with HCQ actually showing more adverse events ("Additionally, 30% of the patients receiving SOC+HCQ reported adverse outcomes compared to 8.8% of patients receiving only SOC, with the most common adverse outcome in the SOC+HCQ group being diarrhea (10% vs. 0% in the SOC group, p=0.004). ") So I was wondering if we should restate the concluding sentence - right now it makes it seem like the study had design flaws but showed some positive associations for HCQ.

**Comment by SiminaB** [comment]
I committed most of the changes that didn't include tagging of other folks... @rando2 do you want me to also commit the changes that have notes for yourself (like checking the build for the citations) or not yet? I need to look over a couple of the other ones tomorrow.

File: content/20.pharmaceuticals.md (hunk 7 above)
Comment (nilswellhausen) [reply content/20.pharmaceuticals.md:689]: As I said this section is outdated by now. I will try to get to this ASAP. Recombinantly expressed neutralizing antibodies such as REGN-COV2 (what former Pres. Trump got) are monoclonal. I think I need to do a better job saying that I focus on therapeutic antibodies and not the neutralizing antibodies that a person developes during an ongoing infection

File: content/20.pharmaceuticals.md (hunk 8 above)
Comment (nilswellhausen) [reply content/20.pharmaceuticals.md:773]: COVID life cycle and drugs (1)

File: content/20.pharmaceuticals.md [hunk 14]
```diff
@@ -30,7 +31,7 @@ However, as of early 2021, COVID-19 remains a significant worldwide concern (Fig
...
 Both approaches hold valuable potential for controlling the impact of the disease.
 Prophylactics bolster immunity to prevent an individual from contracting a disease, whereas therapeutics treat a disease in individuals who have already been infected.
-While vaccine development programs have attracted significant attention and produced a number of promising candidates, there is also an immediate need for treatments that palliate symptoms and prevent the most severe outcomes from infection.
+While a number of vaccines have recently been developed, approved, or are currently being evaluated by the US Food and Drug Administration and other regional and international bodies, vaccination programs only started being rolled out at the very end of 2020 and beginning of 2021, meaning that treatments that palliate symptoms and prevent the most severe outcomes have been necessary for much of 2020 and will continue to be necessary for the foreseeable future.<!--Cite our vaccine manuscript here-->
```
Comment (rando2) [inline content/20.pharmaceuticals.md:34]:
```suggestion
While a number of vaccines have recently been developed, approved, or are currently being evaluated by the US Food and Drug Administration and other regional and international bodies, vaccination programs only started being rolled out at the very end of 2020 and beginning of 2021, meaning that treatments that palliate symptoms and prevent the most severe outcomes have been necessary for much of 2020 and will continue to be necessary for the foreseeable future [@url:https://github.com/greenelab/covid19-review/blob/master/content/22.vaccines.md].
```

File: content/20.pharmaceuticals.md [hunk 15]
```diff
@@ -48,47 +50,48 @@ Similarly, the first case of SARS was reported in November 2002 in the Guangdong
...
+The purpose of this review is to critically appraise the literature surrounding a subset of clinical trials and to evaluate a range of approaches to repurpose existing or develop novel approaches to the mitigation and treatment of coronavirus infections.
+The treatments evaluated here are classified according to their biological properties, specifically whether they are biologics (produced from components of organisms) or small molecules.
 Small molecule drugs include drugs targeted at viral particles, drugs targeted at host proteins, and broad spectrum pharmaceuticals, while biologics include antibodies and interferons.
-As results become available from additional clinical trials, we will continue to update this manuscript to keep pace with the current understanding of which therapeutics may be effective against SARS-CoV-2 or for COVID-19.
+As we cover vaccines in a separate manuscript <!--cite it when available-->, we only consider therapies for the prevention of SARS-CoV-2 infection or COVID-19 in a very limited context here, for example when a drug is studied to see whether it can prevent infection or disease in recently-exposed individuals.
```
[the remark this answers is not in the document]
Comment (rando2) [reply content/20.pharmaceuticals.md:91]: Thanks Ronan! I updated that one too.

File: content/20.pharmaceuticals.md [hunk 16]
```diff
@@ -147,37 +148,35 @@ It should be noted that although the control patients received antivirals, two s
...
 On the other hand, another trial of 60 patients reported a significant effect of favipiravir on viral clearance at four days (a secondary endpoint), but not at 10 days (the primary endpoint) [@doi:10.1093/cid/ciaa1176].
 This study, as well as a prior study of favipiravir [@doi:10/ftgm], also reported that the drug was generally well-tolerated.
 Thus, in combination, these small studies suggest that the effects of favipiravir as a treatment for COVID-19 cannot be determined based on the available evidence, but additionally, none raise major concerns about the safety profile of the drug.
 
 ###### Remdesivir
 
-Remdesivir (GS-5734) is an intravenous antiviral that was developed by Gilead Sciences to treat Ebola Virus Disease (EVD).
+Remdesivir (GS-5734) is an intravenous antiviral that was developed by Gilead Sciences to treat Ebola virus disease (EVD). <!--Seems like it was developed against Hep-C and RSV, but wasn’t effective for them, then it was repurposed for Ebola eg https://www.nih.gov/news-events/nih-research-matters/final-report-confirms-remdesivir-benefits-covid-19 -->
```
Comment (rando2) [inline content/20.pharmaceuticals.md:173]:
```suggestion
Remdesivir (GS-5734) is an intravenous antiviral that was proposed by Gilead Sciences as a possible treatment for Ebola virus disease (EVD). <!--Seems like it was developed against Hep-C and RSV, but wasn’t effective for them, then it was repurposed for Ebola eg https://www.nih.gov/news-events/nih-research-matters/final-report-confirms-remdesivir-benefits-covid-19 -->
```

File: content/20.pharmaceuticals.md (hunk 12 above)
[the remark this answers is not in the document]
Comment (rando2) [reply content/20.pharmaceuticals.md:230]:
```suggestion
```
I can just remove the "Furthermore!"

File: content/20.pharmaceuticals.md (hunk 7 above)
Comment (rando2) [reply content/20.pharmaceuticals.md:689]: Thanks @nilswellhausen! I truly know nothing about this topic, I think I learned a little background for when I was reading through your text last year, but that's it!

File: content/20.pharmaceuticals.md [hunk 17]
```diff
@@ -335,6 +334,7 @@ Therefore, despite the promising potential mechanism, initial results have not p
 
 CQ and hydroxychloroquine (HCQ) are lysosomotropic agents, meaning they are weak bases that can pass through the plasma membrane.
 Both drugs increase cellular pH by accumulating in their protonated form inside lysosomes [@doi:10.1186/1743-422X-8-163; @doi:10.1111/j.1529-8019.2007.00131.x].
+These drugs are used for the treatment and prophylaxis of malaria, as well as the treatment of lupus erythematosus and rheumatoid arthritis in adults [@url:https://dailymed.nlm.nih.gov/dailymed/drugInfo.cfm?setid=34496b43-05a2-45fb-a769-52b12e099341].<!--To Do: Check citation built-->
```
Comment (rando2) [inline content/20.pharmaceuticals.md:337]:
```suggestion
These drugs are used for the treatment and prophylaxis of malaria, as well as the treatment of lupus erythematosus and rheumatoid arthritis in adults [@url:https://dailymed.nlm.nih.gov/dailymed/drugInfo.cfm?setid=34496b43-05a2-45fb-a769-52b12e099341].<!--To Do: Add manual reference-->
```
Does not work!

File: content/20.pharmaceuticals.md [hunk 18]
```diff
@@ -378,7 +379,7 @@ On April 10, 2020, a randomized, non-placebo trial of 62 COVID-19 patients at th
...
 This trial found HCQ decreased both average time to fever break and average time to cough relief, defined as mild or no cough.
 While this study improved on some of the methodological flaws in Gautret et al. by randomizing patients, it also had several flaws in trial design and data analysis that prevent generalization of the results.
-These weaknesses include the lack of placebo, lack of correction for multiple primary outcomes, inappropriate choice of outcomes, lack of sufficient detail to understand analysis, drastic disparities between pre-registration and published protocol, and small sample size.
+These weaknesses include the lack of placebo, lack of correction for multiple primary outcomes, inappropriate choice of outcomes, lack of sufficient detail to understand analysis, drastic disparities between pre-registration [@clinicaltrials:ChiCTR2000029559] and published protocol (including differences in the inclusion and exclusion criteria, the number of experimental groups, the number of patients enrolled, and the in the outcome analyzed), and small sample size.<!--To Do: Do Chinese clinical trial registry ids work? make sure this works at build.-->
```
Comment (rando2) [inline content/20.pharmaceuticals.md:382]:
```suggestion
These weaknesses include the lack of placebo, lack of correction for multiple primary outcomes, inappropriate choice of outcomes, lack of sufficient detail to understand analysis, drastic disparities between pre-registration [@url:http://www.chictr.org.cn/showprojen.aspx?proj=48880] and published protocol (including differences in the inclusion and exclusion criteria, the number of experimental groups, the number of patients enrolled, and the in the outcome analyzed), and small sample size.<!--To Do: Add manual reference.-->
```

File: content/20.pharmaceuticals.md [hunk 19]
```diff
@@ -520,10 +521,10 @@ Several countries have now removed CQ/HCQ from their SOC for COVID-19 due to the
 ##### Dexamethasone
 
 Dexamethasone (9&alpha;-fluoro-16&alpha;-methylprednisolone) is a synthetic corticosteroid that binds to glucocorticoid receptors [@doi:10.1021/cr068203e; @url:https://www.cebm.net/covid-19/dexamethasone/].
-It was first synthesized in the late 1950s as an anti-inflammatory and has been used to treat RA and other inflammatory conditions [@doi:10.1021/ja01545a061; @doi:10.1001/jama.1960.03030070009002].
+It was first synthesized in the late 1950s as an anti-inflammatory and has been used to treat RA and other inflammatory conditions [@doi:10.1021/ja01545a061; @doi:10.1001/jama.1960.03030070009002], including allergies and asthma [@url:https://dailymed.nlm.nih.gov/dailymed/drugInfo.cfm?setid=537b424a-3e07-4c81-978c-1ad99014032a]. <!--Should also mention allergies and asthma. It actually has a wide number of listed uses on the label at https://dailymed.nlm.nih.gov/dailymed/drugInfo.cfm?setid=537b424a-3e07-4c81-978c-1ad99014032a (Under Indications and usage)--><!--To Do: Check reference built correctly-->
```
Comment (rando2) [inline content/20.pharmaceuticals.md:524]:
```suggestion
It was first synthesized in the late 1950s as an anti-inflammatory and has been used to treat RA and other inflammatory conditions [@doi:10.1021/ja01545a061; @doi:10.1001/jama.1960.03030070009002], including allergies and asthma [@url:https://dailymed.nlm.nih.gov/dailymed/drugInfo.cfm?setid=537b424a-3e07-4c81-978c-1ad99014032a]. <!--Should also mention allergies and asthma. It actually has a wide number of listed uses on the label at https://dailymed.nlm.nih.gov/dailymed/drugInfo.cfm?setid=537b424a-3e07-4c81-978c-1ad99014032a (Under Indications and usage)--><!--To Do: Add manual ref-->
```

File: content/20.pharmaceuticals.md [hunk 20]
```diff
@@ -585,7 +586,8 @@ As a result, biologics are another possible avenue through which the pharmacolog
 
 TCZ is a receptor antibody that was developed to manage chronic inflammation caused by the continuous synthesis of the cytokine IL-6 [@doi:10.1101/cshperspect.a016295].
 IL-6 is a pro-inflammatory cytokine belonging to the interleukin family, which is comprised by immune system regulators that are primarily responsible for immune cell differentiation.
-Often used to treat conditions such as RA [@doi:10.1101/cshperspect.a016295], TCZ has become a pharmaceutical of interest for the treatment of COVID-19 because of the role IL-6 plays in this disease. 
+Often used to treat conditions such as RA [@doi:10.1101/cshperspect.a016295], TCZ has become a pharmaceutical of interest for the treatment of COVID-19 because of the role IL-6 plays in this disease.
+It has also been approved to treat CRS caused by CAR-T treatments [@url:https://dailymed.nlm.nih.gov/dailymed/drugInfo.cfm?setid=2e5365ff-cb2a-4b16-b2c7-e35c6bf2de13].<!-- To Do: check reference built-->
```
Comment (rando2) [inline content/20.pharmaceuticals.md:590]:
```suggestion
It has also been approved to treat CRS caused by CAR-T treatments [@url:https://dailymed.nlm.nih.gov/dailymed/drugInfo.cfm?setid=2e5365ff-cb2a-4b16-b2c7-e35c6bf2de13].<!-- To Do: add manual ref-->
```

**Review by rando2 — COMMENTED** [review]
Several references need to be added as manual refs, but probably easier to do on a separate PR. Marking with comments for now.

File: content/20.pharmaceuticals.md [hunk 21]
```diff
@@ -557,12 +558,12 @@ Additionally, dexamethasone is a widely available and affordable medication, rai
 
 The results of the RECOVERY trial's analysis of dexamethasone suggest that this therapeutic is effective primarily in patients who had been experiencing symptoms for at least seven days and patients who were not breathing independently [@doi:10.1056/NEJMoa2021436].
 A meta-analysis that evaluated the results of the RECOVERY trial alongside trials of other corticosteroids, such as hydrocortisone, similarly concluded that corticosteroids may be beneficial to patients with severe COVID-19 who are receiving oxygen supplementation [@doi:10.1053/j.jvca.2020.11.057].
-Thus, it seems likely that dexamethasone is useful for treating inflammation associated with immunopathy or cytokine release syndrome (CRS).
+Thus, it seems likely that dexamethasone is useful for treating inflammation associated with immunopathy or cytokine release syndrome (CRS), which is a condition caused by detrimental hyperactivation of the immune system [@individual-pathogenesis].
```
Comment (rando2) [inline content/20.pharmaceuticals.md:561]:
```suggestion
Thus, it seems likely that dexamethasone is useful for treating inflammation associated with immunopathy or cytokine release syndrome (CRS), which is a condition caused by detrimental overactivation of the immune system [@individual-pathogenesis].
```

File: content/20.pharmaceuticals.md (hunk 17 above)
Comment (agitter) [reply content/20.pharmaceuticals.md:337]: I can go through and create the manual references after this is merged.  I'll search for comments with "manual reference" like you've been adding here.

File: content/20.pharmaceuticals.md (hunk 8 above)
Comment (SiminaB) [reply content/20.pharmaceuticals.md:775]: Great! It seems like step "9" should be "8" though. Also, the circles around 5-7 are much bigger than the others (that's not a big deal, so we can probably fix it in the revision.)

File: content/20.pharmaceuticals.md (hunk 12 above)
[the remark this answers is not in the document]
Comment (SiminaB) [reply content/20.pharmaceuticals.md:230]: Sorry about that! Not sure how it got in!

File: content/20.pharmaceuticals.md (hunk 8 above)
Comment (nilswellhausen) [reply content/20.pharmaceuticals.md:774]: oh wow I just saw that the circles just got all messed up. Let me fix that really quick. Not sure what happened there as it looked normal on my end.

File: content/20.pharmaceuticals.md (hunk 11 above)
Comment (SiminaB) [reply content/20.pharmaceuticals.md:840]: So do we want to keep it or no? Either way is fine with me!

File: content/20.pharmaceuticals.md (hunk 8 above)
Comment (nilswellhausen) [reply content/20.pharmaceuticals.md:774]: COVID life cycle and drugs (2)

Comment (SiminaB) [reply content/20.pharmaceuticals.md:774]: Thank you! Looks awesome!

File: content/20.pharmaceuticals.md (hunk 11 above)
Comment (RLordan) [reply content/20.pharmaceuticals.md:840]: I always feel weird ending a review without some sort of concluding part, but I will let this one up to you folks :)

File: content/20.pharmaceuticals.md (hunk 13 above)
[the remark this answers is not in the document]
Comment (rando2) [reply content/20.pharmaceuticals.md:436]: Ah! Gotcha! Sorry for not understanding! How about this:
```suggestion
Therefore, despite some potential areas of investigation identified in _post hoc_ analysis, this study cannot be interpreted as providing support for HCQ as a therapeutic against COVID-19.
This study provided no support for HCQ against COVID-19, as there was no difference between the two groups in either negative seroconversion at 28 days or symptom alleviation, and in fact, more severe adverse outcomes were reported in the group receiving HCQ.
```

File: content/20.pharmaceuticals.md (hunk 11 above)
Comment (rando2) [reply content/20.pharmaceuticals.md:840]: I think I feel the same as @RLordan -- leave it for now, but maybe we can add the table here to give it more teeth?

File: content/20.pharmaceuticals.md [hunk 22]
```diff
@@ -240,7 +241,7 @@ It reported no significant differences across several outcomes for patients rece
 The data did suggest that the 10-day course might reduce mortality in the most severe patients at day 14, but the representation of this group in the study population was too low to justify any conclusions [@doi:10.1056/NEJMoa2015301].
 Thus, additional research is also required to determine whether the dosage and duration of remdesivir administration influences outcomes.
 
-In summary, remdesivir is a first in class drug due to its FDA approval.
+In summary, remdesivir is a first in class drug due to its FDA approval. <!--It seems like “first in class” means it’s a novel mechanism. Not sure this is clear given the rest of the section-->
```
[the remark this answers is not in the document]
Comment (rando2) [reply content/20.pharmaceuticals.md:246]:
```suggestion
In summary, remdesivir is the first FDA approved anti-viral against SARS-CoV-2 as well as the first FDA approved COVID-19 treatment.
```
Based on feedback from Jeff!
well novel in that it was the

**Review by rando2 — APPROVED** [review]
Thank you so much again for all this feedback @SiminaB! If it sounds okay, I'll commit the last few changes, add the figure to this branch, and merge it in 🎉
